webui/tests/cypress/support/commands.js

```diff
@@ -2,10 +2,22 @@
 // For examples of custom commands please read more here:
 // https://on.cypress.io/custom-commands
 // ***********************************************
-import { ACCOUNT_PASSWORD, ACCOUNT_USERNAME, API_PATH } from '../constants';
+import { ACCOUNT_PASSWORD, ACCOUNT_USERNAME, API_PATH, STORAGE_KEY_AUTH } from '../constants';
 
 const sha512 = require('js-sha512').sha512;
 
+const saveAuthToken = (token) => {
+  return window.localStorage.setItem(STORAGE_KEY_AUTH, JSON.stringify(token));
+};
+
+const retreiveAuthToken = () => {
+  return JSON.parse(window.localStorage.getItem(STORAGE_KEY_AUTH));
+};
+
+const removeAuthToken = () => {
+  window.localStorage.removeItem(STORAGE_KEY_AUTH);
+};
+
```

Comment:
**question:** will [cypress clearing `localStorage` before each test](https://docs.cypress.io/api/commands/clearlocalstorage.html) be a problem for these sets of functions? I think we are aware of it and so we avoid relying on local storage to persist between each test. Maybe it would help to put a comment here to mention cypress's behavior of clearing local storage?

Reply:
> will cypress clearing localStorage before each test be a problem for these sets of functions?

yes it did become an issue that's why all these changes to the `tests` module came to be otherwise the only changes would have been the ones in `support/command.js`

> Maybe it would help to put a comment here to mention cypress's behavior of clearing local storage?

hmm this is not a new behavior added or changed in this PR, it has always excited but our tests just hadn't cared about it yet. So if we wanna leave a comment about this it would be about educating/warning about this general behavior. where would you put that? readme? :thinking: